packages/odyssey-storybook/src/guidelines/DocsStatus.stories.mdx

```diff
@@ -0,0 +1,233 @@
+- Field
+- Field Group
+- Form
+- Heading
+- Icon
+- List
+- Screen Reader Text
+- Text Area (previously with Text Input
+
+#### Guidelines
+
+The following guidelines have not been migrated:
+
+- Color
+- Design Tokens
+- Elements
+- Typography
+- Iconography
+- Updates
+- Help
+- Figma link
+
+### Broken
+
+- Links need to be rewritten throughout.
+
+### Open Questions
+
+- How do we want to style internal reference tables? Using `<table>` for SB styling? Using `Table` for React dogfood?
+- How do we author stories for Bad Examples?
+- Should we have stories for all available optional-part combinations?
+  - If not, what do we document and test visually?
+- What should the default tab be? What should that behavior look like? Test `viewMode: 'docs'`
+- Story style is inconsistent between example content and descriptive content. Rec: example content
+- How do we best document browser states like `:hover` for the user?
+  - Should these require interaction?
+  - What about `:disabled`?
+- Related, how do we best document stories re: grammar or word choice?
+- How should we order content sections?
+
+## Components
+
+### Banner
+
+#### Stories
+
+The existing Banner stories don't make distinctions regarding the optional and mandatory elements, so all stories currently include all features.
+
+**Recommendation:** Rewrite existing stories to showcase minimal features with additional stories showcasing Title, Dismiss, and Actions.
+
+##### Existing
+
+- Info
+- Danger
+- Caution
+- Dismissable (identical to Info)
+
+##### Assumed
+
+###### Variants
+
+- Info
+- Danger
+- Caution
+
+###### Features
+
+- Title
+- Actions
+- Dismissable
+
+### Button
+
+#### Stories
+
+We are missing browser-state stories for Button (e.g. `:hover`, `:focus`, `:disabled`).
+
+- Should these examples be provided in the docs for each component with browser states?
+- Are these stories? Something else?
+
+We're missing stories for Good/Bad usage examples as well as content-specific examples (e.g. capitalization, word choice).
+
+### Checkbox
+
+#### Stories
+
+- Checkbox stories always include error text, which is styled incorrectly.
+- Similar issue with Button re: browser state examples.
+- Checkbox stories default to invalid due to styling.
+
+### Field
+
+The docs for Field have been migrated over from Field Label. Field, FieldGroup, and Form all need net new documentation.
+
+This older deployment offers a starting point: https://9d6b542.ods.dev/components/form/
+
+### Icon
+
+Previously, Icon was not a component, so those "guideline" docs have not been migrated.
+
+### Infobox
+
+#### Stories
+
+Similar to Banner, Infobox stories surface all options up front, with no visual examples for optional components like Actions. Nor do they surface alternate approaches for Actions (inline action vs actions section).
+
+We're also missing examples for content guidelines.
+
+### Link
+
+#### Documentation
+
+We're missing documentation for the Secondary link variant.
```

Comment:
Did this just not exist before?

Reply:
Yup. It looks like some of the updates from this year didn't get accompanying docs changes. :(